app/components/pages/components/base.js

```diff
@@ -0,0 +1,7 @@
+export default ({openModal}) =>
```

Comment:
This is anal, but I prefer if props/function references are more descriptive/explicit. In this case, it'd be better to not share the same name as the `openModal` action, but rather be called `openExampleModal` or whatever it's called.

Does this make sense? My ability to communicate is quickly deteriorating today.

Reply:
yeah makes sense and not really anal